apis/python/tests/test_basic_anndata_io.py

```diff
@@ -472,6 +472,85 @@ def test_add_matrix_to_collection(conftest_pbmc_small, tmp_path):
         assert sorted(list(exp_r.ms["RNA"]["newthing"].keys())) == ["X_pcd"]
 
 
+@pytest.mark.parametrize(
```

Comment:
there do not appear to be any tests with pre-1.15 (old-style shape) Experiments.

Is that explicitly unsupported, or should we add tests?  The implementation in ingest.py seems to support it, so I suggest testing.  There is sample data already available to make this really easy -- see `test_soma_experiment_versions.py` for an example test that uses many old variants for complete coverage